python-lib/sharepoint_client.py

```diff
@@ -308,6 +308,8 @@ def create_path(self, file_full_path):
                 logger.error("Could not create folder for '{}'. Check your write permission for the folder {}.".format(path, previous_path))
 
     def move_file(self, full_from_path, full_to_path):
+        assert_no_percent_in_path(full_from_path)
+        assert_no_percent_in_path(full_to_path)
```

Comment:
Nit: might be better to put these in SharePointFSProvider with the other asserts? (But you can probably argue it both ways...)

Reply:
Done in [1cbae10](https://github.com/dataiku/dss-plugin-sharepoint-online/pull/66/commits/1cbae10d09cebebdf80606da092842e18eb9c79a)